cumulus/tasks/delete-ingest-tracking-data/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "delete-ingest-tracking-data",
```

Comment:
could you rename this `@cumulus/delete-ingest-tracking-data`? we have the `@cumulus` in front of all the names to make them publishable to npm

Reply:
will do.  Thanks.